DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -425,6 +425,14 @@ function SetWatchFlags() {
             }
         }
         bt = class'BingoTrigger'.static.Create(self,'CrackSafe',vectm(0,0,0));
+
+        bt = class'BingoTrigger'.static.Create(self,'WarehouseSewerTunnel',vectm(-1348.603027, 211.117294, -459.900452),80,60);
+        bt.bDestroyOthers = false;
```

Comment:
why not bDestroyOthers? this seems like something you should only be able to do once